src/test/java/com/microsoft/sqlserver/jdbc/TestResource.java

```diff
@@ -201,6 +201,7 @@ protected Object[][] getContents() {
             {"R_objectNullOrEmpty", "The {0} is null or empty."},
             {"R_cekDecryptionFailed", "Failed to decrypt a column encryption key using key store provider: {0}."},
             {"R_connectTimedOut", "connect timed out"},
+            {"R_AccessTokenCallbackWithUserPassword", "Cannot set the access token callback if \"User\", \"UserName\" or \"Password\" has been set."},
```

Comment:
I don't think we need to add this, this is the same as in SQLServerResource, just use e.getMessage().matches(TestUtils.formatErrorMsg(R_AccessTokenCallbackWithUserPassword))